tensorflow/tools/docker/jupyter_notebook_config.py

```diff
@@ -16,7 +16,7 @@
 from IPython.lib import passwd
 
 c.NotebookApp.ip = '*'
-c.NotebookApp.port = 8888
+c.NotebookApp.port = int(os.getenv('PORT', '') or 8888)
```

Comment:
@craigcitro would you like me to change it to int(os.getenv('PORT', 8888))? I don't have a strong preference, actually your proposal seems cleaner to me.